swagger.yaml

```diff
@@ -18,7 +18,13 @@ schemes:
 parameters:
   instanceId:
     name: instanceId
-    description: "A unique instanceId that relates to an import job"
```

Comment:
We shouldn't leave this description as an empty string, how about: `A unique instanceId that corresponds to one of the possible several instances that make up a job` ?